operator-kustomize/deployment.yaml

```diff
@@ -21,7 +21,10 @@ spec:
           resources:
             requests:
               memory: 256Mi
-          securityContext:
-            runAsUser: 1000
-            runAsNonRoot: true
-            readOnlyRootFilesystem: true
+          volumeMounts:
+            - mountPath: /var/run/docker.sock
+              name: docker-access
+      volumes:
+        - name: docker-access
```

Comment:
This assumes docker sock is available. Not sure if this is true for all our customers.